tox.ini

```diff
@@ -1,33 +1,19 @@
 [tox]
 envlist =
-    py27,
-    py33,
-    py34,
-    py35,
-    py36,
-    pypy,
-    pypy3,
-    coverage-report,
+    py27,pypy,py34,py35,py36,pypy3,coverage
 
 [testenv]
 commands =
-    coverage run -m zope.testrunner --test-path=src {posargs:-vc}
+    zope-testrunner --test-path=src []
 deps =
     .[test]
-    zope.testrunner
-    coverage
-setenv =
-    COVERAGE_FILE=.coverage.{envname}
 
-[testenv:coverage-report]
-basepython = python2.7
-deps = coverage
-setenv =
-    COVERAGE_FILE=.coverage
-skip_install = true
+
+[testenv:coverage]
```

Comment:
It does slow the tests down noticeably under PyPy.

But this was mostly about standardization---the vast majority of the other tox.ini I've been working on have a separate coverage environment, so that's what I did here. When we can achieve 100% coverage in any given Python environment (so that we don't have to combine multiple runs) that seems easier to me.